lib/qif2json.js

```diff
@@ -59,7 +81,10 @@ exports.parse = function(qif, options) {
         }
         switch (line[0]) {
             case 'D':
-                transaction.date = parseDate(line.substring(1), options.dateFormat);
+                transaction.date = parseDate(line.substring(1), options.usDates, options.oldDates);
```

Comment:
`dateFromat` was pretty flexible before - there was only one option used yes, but it could be extended. Splitting it into 2 separate flags gives us fewer options going forward (ie we just have to keep adding more booleans for new date formats)

Could oldDates be supported with another dateFormat instead? `pre-y2k` maybe?

Reply:
Then you're parsing a string or something if you want both options (which I do), or having a "if prey2k-us, or us" or whatever. If anything I think it'd be cleaner just to pass a dateFormat object or something.

Reply:
Maybe an array of date formats? I can imagine that across the various implementations of the QIF format there are many different possible date formats, so having something extensible like a format string (or list of format strings) is probably the most flexible?